tests/configurations/jest.teardown.js

```diff
@@ -0,0 +1,5 @@
+const rimraf = require('rimraf');
+
+module.exports = function () {
+  rimraf.sync('/tmp/token-cli-test-workdir');
```

Comment:
the value in jest.setup.js and jest.teardown.js should be the same, extract it to a common file.

Reply:
extracted